hasher-matcher-actioner/webapp/src/Upload.js

```diff
@@ -50,18 +66,15 @@ export default function Upload() {
           </Collapse>
           <Collapse in={submitted}>
             <div>
-              <p>Hash created for {fileName}: acecf3355e3125c8e24e2f30e0d4ec4f8482b878b3c34cdbdf063278db275992</p>
-              <p>...either this...</p>
-              <p>{fileName} does not match any signals.</p>
-              <p>...or this...</p>
-              <p>{fileName} matches one or more signals.</p>
+              <p>Uploaded {fileName}: hash should be generated shortly... </p>
```

Comment:
Just calling out this bit needs some kind of polling to check for the hash and match(es). What do you think about two subsequent APIs here, one to check for a hash and one for matches? As they respond, the UI's content would change as follows:

1. first, only call the api looking up the hash; show the message at 69 as is until that api returns info. Assume polling every few seconds is OK for this.
2. next, after we have the hash, start polling for matches. Initially show "Checking for matches..." as the message. Once the api returns something, show information about the matches. If we poll a number of times such that the polling has happened for 60s (120s?) show "No match(es) found."